packages/flame/lib/src/game/mixins/game.dart

```diff
@@ -95,6 +95,8 @@ mixin Game on Loadable {
   /// Use for calculating the FPS.
   void onTimingsCallback(List<FrameTiming> timings) {}
 
+  void onMount() {}
```

Comment:
I don't think `onMount` and `onRemove` is needed in this mixin since it is meant for games that live can live in the component tree, for other games `onAttach` and `onDetach` can be used.

Reply:
Ok, great, I'll remove them then.

Reply:
I just tried removing them, and it turns out to be a breaking change -- even 2 tests start to fail. 
So, probably such breaking change should be in its own PR.